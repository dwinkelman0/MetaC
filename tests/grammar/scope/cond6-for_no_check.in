{
    for (const char *it = str; ; it += 1) {
        if (!*it) {
            break;
        }
        printf("%c", *it);
    }
    printf("\n");
}