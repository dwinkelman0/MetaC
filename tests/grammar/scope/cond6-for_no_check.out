{
    for (const char *it = str; 1; it += 1) {
        if (!*it) {
            break;
        }
        printf("%c", *it);
    }
    printf("\n");
}